test/appium/tests/atomic/chats/test_public.py

```diff
@@ -155,3 +155,20 @@ def test_user_can_interact_with_public_chat(self):
 
         self.verify_no_errors()
 
+    @marks.testrail_id(5675)
+    @marks.high
+    def test_redirect_to_public_chat_tapping_tag_message(self):
+        signin = SignInView(self.driver)
+        home_view = signin.create_user()
+        chat = home_view.join_public_chat('montagne-angerufen')
+        tag_message = '#spectentur'
```

Comment:
Chat history is fetched only for 24 hours, the test will not get the message each time. Could you add a step with sending the message each time?

Reply:
> Chat history is fetched only for 24 hours, the test will not get the message each time. Could you add a step with sending the message each time?

please check it out